etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -203,7 +193,7 @@ spec:
           value:  {{.Values.global.noProxy}}
         {{ end }}
         - name: ENABLE_WORKER_SECURITY_CONTEXTS
-          value: {{ .Values.pachd.securityContext.enabled | quote }}
+          value: {{ .Values.global.securityContexts.enabled | quote }}
```

Comment:
I think some of the changes you're making here should propagate to the worker RCs; src/server/pps/server/worker_rc.go line 332 (sidecar) and line 348 (user code) is where those are defined.